src/middleware/src/storage/src/indexeddb.js

```diff
@@ -273,28 +273,31 @@ export default class IndexedDB {
     });
   }
 
-  static isSupported() {
-    const name = 'testIndexedDBSupport';
+  static loadAdapter(name) {
     const indexedDB = global.indexedDB || global.webkitIndexedDB || global.mozIndexedDB || global.msIndexedDB;
+    const db = new IndexedDB(name);
 
-    if (typeof indexedDB === 'undefined') {
-      return Promise.resolve(false);
+    if (typeof isSupported !== 'undefined') {
+      if (isSupported) {
+        return Promise.resolve(db);
+      }
+
+      return Promise.resolve(undefined);
     }
 
-    if (typeof isSupported !== 'undefined') {
-      return Promise.resolve(isSupported);
+    if (typeof indexedDB === 'undefined') {
+      isSupported = false;
+      return Promise.resolve(undefined);
     }
 
-    const db = new IndexedDB(name);
-    return db.save(name, { _id: '1' })
-      .then(() => db.clear())
+    return db.save('__testSupport', { _id: '1' })
```

Comment:
Sorry, my initial question wasn't clear. I was asking about the change of name from `testIndexedDBSupport` to `__testSupport`. Whether that change will result in 2 entries for older Kinvey apps (one for each name).
I don't think it's a big deal, but it's worth noting that the change potentially affects older clients.